core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -139,6 +139,8 @@ object StorageTool extends Logging {
     if (namespace.getBoolean("standalone")) {
       formatter.setInitialVoters(createStandaloneDynamicVoters(config))
     }
+    Option(namespace.getList("add_scram")).
```

Comment:
I had to double-check this since the no-scram case is actually a null rather than an empty list... 🫤